Makefile

```diff
@@ -316,7 +316,7 @@ frontend-build-with-profiler:
 
 .PHONY: frontend-fast
 frontend-fast:
-	cd frontend/ ; yarn workspace @streamlit/app buildFast
```

Comment:
This no longer exists, so I updated it.

Reply:
Do we even still need `frontend-fast`? Is it faster compared to `make frontend`?